### DISCUSSION TITLE
杀戮尖塔 正式版|官方中文|V02.12.2022|解压即撸|

### DISCUSSION THREAD

**Discussion by match-fire** [share121/inter-knot#1587] [其他]
![](https://shared.st.dl.eccdnx.com/store_item_assets/steam/apps/646570/header.jpg?t=1580430702)
名称: Slay the Spire

你可以通过如下操作将游戏语言设置为简体中文：点击游戏菜单中的“Settings”选项，在设置窗口中将右下角的“Language”从“English”改为“Chinese (Simplified)”，然后重新启动游戏。
我们将卡牌游戏与Roguelike游戏融为一体，制作出了这款最棒的单机组牌游戏。打造出独一无二的牌组、遇见无数离奇的生物、发现威力强大的遗物、去屠戮这座高塔吧！

游戏特色
动态构建的牌组：谨慎选择你的卡牌！在攀爬高塔的途中邂逅百余张不同的卡牌，选择是否将它们加入你的牌组，通过组合和搭配来让牌组发挥力量，打败敌人、挑战登顶。
不断变化的高塔：每次开始爬塔的旅程时，高塔的构造都会发生变化。是选择高风险高回报，还是谨慎前行？面对不同的敌人、选择不同的卡牌、发现不同的遗物、甚至挑战不同的Boss！
威力强大的遗物：在塔中你将发现名为遗物的各种强大物品，这些遗物可能会与你的牌组产生强大的化学反应，让你的战力得到飞跃。但也不要过于大意，为了获得遗物，有时你所付出的代价，可会超过区区金钱……

![](https://shared.st.dl.eccdnx.com/store_item_assets/steam/apps/646570/extras/store_defect.gif?t=1592339399)
![](https://shared.st.dl.eccdnx.com/store_item_assets/steam/apps/646570/ss_c171816f7ecd35b5b46d2fa27532f4c5b8ca3cc5.jpg?t=1580430702)
![](https://shared.st.dl.eccdnx.com/store_item_assets/steam/apps/646570/ss_01aa3e7759e457bfbf2422f31c325d7b3ba8a6eb.jpg?t=1580430702)
![](https://shared.st.dl.eccdnx.com/store_item_assets/steam/apps/646570/ss_8af73365121b1b1e2d8004fca2cbffb9b9f6d7a5.jpg?t=1580430702)
![](https://shared.st.dl.eccdnx.com/store_item_assets/steam/apps/646570/ss_5b2b32f09da5a5bed99d5c5168b50fd681a4690c.jpg?t=1580430702)

链接: https://pan.baidu.com/s/1zs0k2jDZOOaq8_-WNXZyig

如遇到有带x的提取码请手打输入: ybd4

解压密码：xxxxx520.com

**Comment by Your-O** [comment]
塔？？？你怎么也到这里了？？

**Comment by cpu644** [comment]
塔？？？
